devito/ops/operator.py

```diff
@@ -32,12 +33,11 @@ def __init__(self, *args, **kwargs):
     def _specialize_iet(self, iet, **kwargs):
         warning("The OPS backend is still work-in-progress")
 
-        # If there is no iteration tree, then there is no loop to be optimized using OPS.
-        iteration_tree = retrieve_iteration_tree(iet, mode='normal')
-        if not len(iteration_tree):
+        affine_trees = find_affine_trees(iet).items()
+
+        # If there is no affine trees, then there is no loop to be optimized using OPS.
+        if not affine_trees:
             return iet
```

Comment:
@btrb The iet will not be specialized for backend OPS, but will still call the CompilerOPS, causing an error. How do you want to proceed?

Reply:
I'm proceeding with merging, this can be addressed in another PR